tests/unittests/functionality/test_google_oauth.py

```diff
@@ -21,9 +21,9 @@ def test_user_already_logged_in(self):
             register(self.app, 'email@gmail.com', 'test')
             logout(self.app)
             login(self.app, 'email@gmail.com', 'test')
-            self.assertTrue('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code',
+            self.assertTrue('Open Event' in self.app.get('/gCallback/?state=dummy_state&code=dummy_code&test=1',
```

Comment:
@niranjan94 @SaptakS I'm not sure if this is the right way to pass tests. Can you please provide any suggestion here.

The issue is that I cannot redirect the authenticated user directly to admin index without saving the user details ( as the user presses the connect button ) but the test expect a redirect here.